episodes/05-history.md

```diff
@@ -228,7 +228,7 @@ If we want to go back even further,
 we can use a commit identifier instead:
 
 ```bash
-$ git restore f22b25e mars.txt
+$ git restore -s f22b25e mars.txt
```

Comment:
Then here perhaps we mention that the `-s` flag is needed because we are restoring to a specific source commit.